nepal_compliance/nepal_compliance/doctype/ird_certification/ird_certification.py

```
@@ -0,0 +1,122 @@
+            try:
+                if ext == ".pdf":
+                    merger.append(path)
+                    files_merged += 1
+                elif ext in [".jpg", ".jpeg", ".png"]:
+                    img = Image.open(path).convert("RGB")
+                    img_bytes = io.BytesIO()
+                    img.save(img_bytes, format="PDF")
+                    img_bytes.seek(0)
+                    merger.append(img_bytes)
+                    files_merged += 1
+            except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)